From 23826db3def9d713db098670aba6b80e1ca87baa Mon Sep 17 00:00:00 2001
From: Hongzhang Yang <Hongzhang.Yang@freescale.com>
Date: Fri, 16 Nov 2012 10:03:06 +0800
Subject: [PATCH 221/259] ENGR00233797 Enlarge contextBufMem.size for H.264
 decoder

VPU lib v5.4.8

- Enlarge contextBufMem.size for H.264 decoder to adapt to FW v2.1.8 change
- Enable dprintf on Android
- Rectify address of BIT_RESET_CTRL on i.MX6

Signed-off-by: Hongzhang Yang <Hongzhang.Yang@freescale.com>
---
 vpu/vpu_debug.h | 19 +++++++++++--------
 1 file changed, 11 insertions(+), 8 deletions(-)

diff --git a/vpu/vpu_debug.h b/vpu/vpu_debug.h
index 55461b1..2721755 100644
--- a/vpu/vpu_debug.h
+++ b/vpu/vpu_debug.h
@@ -31,14 +31,6 @@
 extern int vpu_lib_dbg_level;
 
 #ifdef BUILD_FOR_ANDROID
-#define dprintf(level, fmt, arg...)     if (vpu_lib_dbg_level >= level) \
-        LOGD(fmt, ## arg)
-#else
-#define dprintf(level, fmt, arg...)     if (vpu_lib_dbg_level >= level) \
-        printf("[DEBUG]\t%s:%d " fmt, __FILE__, __LINE__, ## arg)
-#endif
-
-#ifdef BUILD_FOR_ANDROID
 #ifndef ALOGE
 #define ALOGE LOGE
 #endif
@@ -48,6 +40,9 @@ extern int vpu_lib_dbg_level;
 #ifndef ALOGI
 #define ALOGI LOGI
 #endif
+#ifndef ALOGD
+#define ALOGD LOGD
+#endif
 #define err_msg		ALOGE
 #define info_msg	ALOGI
 #define warn_msg	ALOGW
@@ -66,6 +61,14 @@ extern int vpu_lib_dbg_level;
 	} while (0)
 #endif
 
+#ifdef BUILD_FOR_ANDROID
+#define dprintf(level, fmt, arg...)     if (vpu_lib_dbg_level >= level) \
+        ALOGD(fmt, ## arg)
+#else
+#define dprintf(level, fmt, arg...)     if (vpu_lib_dbg_level >= level) \
+        printf("[DEBUG]\t%s:%d " fmt, __FILE__, __LINE__, ## arg)
+#endif
+
 #define ENTER_FUNC() dprintf(4, "enter %s()\n", __func__)
 #define EXIT_FUNC() dprintf(4, "exit %s()\n", __func__)
 
-- 
1.8.0

